alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -109,7 +110,8 @@ func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 				Default:  "ADVERTISE_DEFAULT_ROUTE",
 				ValidateFunc: validation.StringInSlice([]string{
 					"ADVERTISE_DEFAULT_ROUTE",
-					"ADVERTISE_CUSTOM_PREFIX"}, false),
+					"ADVERTISE_CUSTOM_PREFIX",
+				}, false),
```

Comment:
same here, this is not really related to what this PR should do?

Reply:
Again, I am not really sure why benefit to have this new format. `gofmt` is built-in and generic enough across all our Golang services. We should stick with that.